pyomo/contrib/appsi/solvers/tests/test_highs_persistent.py

```diff
@@ -18,7 +18,6 @@
 from pyomo.common.log import LoggingIntercept
 from pyomo.common.tee import capture_output
 from pyomo.contrib.appsi.solvers.highs import Highs
-from pyomo.contrib.appsi.base import TerminationCondition
```

Comment:
This import was necessary - the lack of it is what's causing your test failures currently.

Reply:
I am at work right now and unfortunately they have locked down pushes from both personal AND enterprise accounts... I will fix it once home

Reply:
Done